WD-21650 - Refresh /solutions/edge-computing

pedoch opened the pull request:
## Done

- Refresh the /solutions/edge-computing page (previously /edge-computing page).

## QA

- Check out the demo and navigate to /solutions/edge-computing
- Make sure the UI matches the designs here: https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/canonical.com-solutions?node-id=3283-1665&m=dev
- Make sure the content matches the suggestions in the copy doc: https://docs.google.com/document/d/1pWMeqRw3io8DlHhNDq4tZ5EKDvLJbVS4vA1kQthzLMc/edit?tab=t.0

## Issue / Card

- [WD-21650](https://warthogs.atlassian.net/browse/WD-21650)
- [WD-21647](https://warthogs.atlassian.net/browse/WD-21647)

kim-isaac commented:
Thanks @pedoch ,looks great! l have two comments below:

- [ ] On the 'Securely-designed edge infrastructure' section, regular bottom padding is overlapped. could you delete one please?
- [ ] Sorry that I missed logos on the quote should have top padding within the image. Could you change those 4 assets to [these](https://assets.ubuntu.com/manager?tag=solutions-edge-computing-logo&asset_type=&product_types=&file_types=)?

eliman11 commented:
Thanks @pedoch! Some comments -
- [ ] Typo in hero paragraph, we should delete the additional S - "S Scale with ease and keep in.." 
- [ ] Re your comment on the copydoc, "[Read the full case study ›](https://ubuntu.com/engage/secure-digital-signage-screenly)" link under "Healthcare" should be https://canonical.com/case-study/grundium-ubuntu-pro-for-devices instead
- [ ] Could we change the HTML tags for these subheadings in the tabbed section into H3s instead of H2s, whilst keeping styling the same?
<img width="590" alt="Screenshot 2025-05-22 at 11 51 56" src="https://github.com/user-attachments/assets/2c13cd9a-b813-4fb7-ab50-a17cccdffb5a" />

- [ ] Add aria-labels to "Read/learn more ›" links under "Learn more about open source edge computing" section. These labels should be "Read/Learn more about (title of content)" so "Read more about edge computing use cases" for the first link for example.

eliman11 commented:
Sorry one more comment I missed - I just realized we're changing the URL from https://canonical.com/edge-computing to https://canonical.com/solutions/edge-computing. Could you please archive the original page and set up redirects in that case?

pedoch commented:
@kim-isaac I've addressed the comments. Please take another look.

pedoch commented:
@eliman11 I have addressed your comments and added the redirect. Please take another look.

pedoch commented on templates/edge-computing/index.html:
Done!

pedoch commented on secondary-navigation.yaml:
```diff
@@ -30,6 +30,14 @@ ai:
     - title: Big data analytics
       path: /solutions/ai/big-data-ai
 
+edge-computing:
+  title: Edge computing
+  path: /solutions/edge-computing
+
+  children:
+    - title: Overview
+      path: /solutions/edge-computing
+
```
If I take it out of the `secondary-navigation.yaml` file the bubble doesn't get created. How would you advise this gets handled? 

If the children property isn't passed I get a parsing error. I saw other entries with one child (The overview child) so I followed that pattern.

Perhaps we need to go over the logic for parsing this and make sure not to create the dropdown of only the overview child is present, but I think this is out of scope for this PR.

pedoch replied:
I ended up taking out the secondary nav based on a comment on MM.

kim-isaac commented:
Thanks @pedoch for addressing, I have two comments left:

- [ ] I found one typo thing on the hero - please delete comma after 'tooling.'
- [ ] Could we keep a tab on the mobile like this below?

![image](https://github.com/user-attachments/assets/9de12b48-9d8f-4b7e-b476-20d8b748f833)

eliman11 commented:
Thanks @pedoch looks good to me!

kim-isaac commented:
Really appreciate the fast update @pedoch ! I'll add +1

rknipe-canon commented:
Hi all, from a visual perspective, looks great, thanks for working on this. 

- Added a couple of textual amends to the figma and the copydoc, concerning the main page subheading and the Industrial IoT use case. 
- Added an OG image. 
- In terms of navigation, I'd like to revert back to having the page linked from ubuntu.com/cloud.cloud-computing (so no individual bubble for this page) to preserve SEO. 

Aside from that, all good. Thanks for all of your hard work on this.

eliman11 commented:
Sorry just chiming in here to expand on Rhy's comment! For the og:image please use the assets manager link Mattea added (https://assets.ubuntu.com/v1/e213bf98-og-image-2.png) and for navigation there's nothing to be done here as we've set up the redirects :)

pedoch commented:
Just to clarify, @eliman11 @rknipe-canon the page was previously on `/edge-computing`, I moved it to `/solutions/edge-computing` based on the copy doc and set up redirects (I also added an item to the solutions list in the main navbar).

But now the copy docs has been reverted, should I revert the route change I made or leave it the way it is now on `/solutions/edge-computing`
